include/dsn/tool-api/rpc_message.h

```diff
@@ -148,6 +153,41 @@ class message_ex : public ref_counter,
     // by message queuing
     dlink dl;
 
+    // lantency tracer
+    std::unique_ptr<dsn::tool::latency_tracer> tracer;
+
+    void report_tracer(uint64_t time_threshold)
+    {
+        if (tracer == nullptr || !time_threshold) {
+            return;
+        }
+
+        uint64_t total_time_used = tracer->end_time - tracer->start_time;
+        if (total_time_used < time_threshold) {
+            return;
+        }
+        std::string header("read request latency tracer log");
+        std::string log;
+        log = fmt::format("\nTRACER:name={}, request_id={}, "
+                          "start_time={}, request_type=[{}]",
+                          tracer->name,
+                          tracer->id,
+                          tracer->start_time,
+                          tracer->request_type);
+
+        for (const auto &iter : tracer->points) {
```

Comment:
`std::map<std::string, uint64_t> points;` points are ordered by keys, outputs are not ordered by the time added them, is it what you expect?

Reply:
Now the key start with `ts`, for example:
```
ts=1592388860800564738, replication_service_app::on_intercepted_request=13898
ts=1592388860800567803, mutation_queue::add_work=3065
ts=1592388860800578241, prepare_list::prepare=6083
```
it will just ordered by the ts, but I'm ready to use `unordered_map`